source/code/plugin/in_kube_nodes.rb

```diff
@@ -109,7 +110,12 @@ def enumerate
                     dockerVersion.slice! "docker://"
                     containerNodeInventoryRecord['DockerVersion'] = dockerVersion
                     # ContainerNodeInventory data for docker version and operating system.
-                    containerNodeInventoryEventStream.add(emitTime, containerNodeInventoryRecord) if containerNodeInventoryRecord
+                    containerNodeInventoryWrapper = {
```

Comment:
Ignore this change. Since the PR was created before the target branch was updated, this shows up.